examples/without_parent_zone/fixtures.us-west-1.tfvars

```diff
@@ -0,0 +1,7 @@
+region = "us-west-1"
```

Comment:
@kevcube we are using `us-east-2` as the test region

https://github.com/cloudposse/terraform-example-module/blob/main/examples/complete/fixtures.us-east-2.tfvars

would you mind updating it everywhere and changing the file name?